### DISCUSSION TITLE
Help with DHT service

### DISCUSSION THREAD

**Discussion by omeroman0** [RaresAil/homebridge-gree-ac#13] [Q&A]
First of all, great plugin. Best I have tried for my ACs so far. Thank you for your hardwork.

Question is how/where to put the DHT service snippet? I host my HB instance on a pi, so I think I can apply this? Copied your sample to the field in the Config in the UI. So if this works, will it display the real temp, not the default current temp as entered?

Thanks again.

**Comment by RaresAil** [comment]
Hey, so you need to run that js file as a service, because that file opens a local server which reads the dht data, and from the config add the ip of that service which is `http://localhost:55555`.

To run the service search for `how to run a node service in systemd`
